runtime/compiler/control/CompilationThread.cpp

```diff
@@ -8787,13 +8787,20 @@ TR::CompilationInfoPerThreadBase::compile(
                {
                TR_ASSERT(_compiler.getHotnessName(_compiler.getMethodHotness()), "expected to have a hotness string");
                if (_compiler.getOutFile() != NULL && _compiler.getOption(TR_TraceAll))
-                  traceMsg(&_compiler, "<compile\n"
-                          "\tmethod=\"%s\"\n"
-                          "\thotness=\"%s\"\n"
-                          "\tisProfilingCompile=%d>\n",
-                          _compiler.signature(),
-                          _compiler.getHotnessName(_compiler.getMethodHotness()),
-                          _compiler.isProfilingCompilation());
+                  {
```

Comment:
What is printed if we don't have `TR_TraceAll` but have another type of tracing (like traceCG)

Reply:
Currently if we don't have `-Xjit:'{*}(traceFull),log=log` in option, client UID is still printed in compilation log.
Compilation log from `-Xjit:'{*}(traceCG),log=log` option
```
<compile
        method="sun/reflect/Reflection.getCallerClass()Ljava/lang/Class;"
        hotness="cold"
        isProfilingCompile=0
        clientID=7954624449399674535>
</compile>
```

Is that what we wanted? Or shall we only print client UID when `traceFull` is specified?

Reply:
Yes, we want the clientID even when traceFull is not specified.
Is there an extra `>` after the UID?
It would help to give an example on how the output looks before your changes and after your changes

Reply:
I think this `>` is part of the xml syntax
```
<compile
    XXX
    XXX>
</compile>
```

Here is a comparison before & after change running the following command
`./java -XX:+UseJITServer -Xjit:'{*}(traceFull),log=log' -version`

Before change
```
<compile
	method="sun/reflect/Reflection.getCallerClass()Ljava/lang/Class;"
	hotness="cold"
	isProfilingCompile=0>
</compile>
```

After change
```
<compile
        method="sun/reflect/Reflection.getCallerClass()Ljava/lang/Class;"
        hotness="cold"
        isProfilingCompile=0
        clientID=9710225490534608421>
</compile>
```

Reply:
ok